shop/urls.py

```diff
@@ -28,5 +29,10 @@
     path('orders/<int:pk>/', OrderView.as_view(), name='order-detail'),
     path('order-items/', OrderItemView.as_view(), name='order-item-list'),
     path('order-items/<int:pk>/', OrderItemView.as_view(), name='order-item-detail'),
+    path('users/', UserView.as_view(), name='user-list'),
+    path('users/<int:pk>/', UserView.as_view(), name='user-detail'),
+    path('users/<int:pk>/addresses/', UserAddressesView.as_view(), name='user-addresses'),
+    path('users/<int:pk>/feedback/', UserFeedbackView.as_view(), name='user-feedback'),
```

Comment:
feedbacks?

Reply:
Word  feedback have the same word for the plural.